packages/plugin-commands-installation/src/recursive.ts

```diff
@@ -432,5 +432,5 @@ export function matchDependencies (
   if (include.optionalDependencies) {
     allDependencies = allDependencies.concat(Object.keys(manifest.optionalDependencies || {}))
   }
-  return input.filter(i => !i.includes('*')).concat(allDependencies.filter(match))
+  return input.concat(allDependencies.filter(match))
```

Comment:
You may use the `...` syntax instead of concat.

```suggestion
  return [
    ...input,
    ...allDependencies.filter(match),
  ]
```